### PR TITLE
Add a preference to allow Profiles menu to open in new windows by default

### PR DESCRIPTION
**Pull request by zeroimpl**
I find I always want the profile menu to open in a new window, so added an advanced option to swap to this behaviour, instead of having to press option each time.

### CODE REVIEW

File: sources/iTermApplicationDelegate.m [hunk 1]
```diff
@@ -1440,12 +1440,21 @@ - (void)nonTerminalWindowBecameKey:(NSNotification *)aNotification {
...
-    DLog(@"Updating address book menu");
+- (void)updateAddressBookMenu:(NSNotification *)aNotification {
+    DLog(@"Updating Profile menu");
     JournalParams params;
-    params.selector = @selector(newSessionInTabAtIndex:);
+    if ([iTermAdvancedSettingsModel openProfilesInNewWindow]) {
+        params.selector = @selector(newSessionInWindowAtIndex:);
+        params.alternateSelector = @selector(newSessionInTabAtIndex:);
+        [bookmarkMenu itemAtIndex:2].title = [[bookmarkMenu itemAtIndex:2].title stringByReplacingOccurrencesOfString:@"Window" withString:@"Tab"];
+        [bookmarkMenu itemAtIndex:3].title = [[bookmarkMenu itemAtIndex:3].title stringByReplacingOccurrencesOfString:@"Window" withString:@"Tab"];
```
Comment (gnachman) [inline sources/iTermApplicationDelegate.m]: I think you'll also want to update the last item, which says "Open All in New Window" when holding option.

Comment (zeroimpl) [reply sources/iTermApplicationDelegate.m]: So right now, that item still works the old way - will do tabs unless you hold Option.  Think it makes sense to flip it as well?

Comment (gnachman) [reply sources/iTermApplicationDelegate.m]: After taking a second look, I think it's OK to keep it as is. I'll merge this.

**Comment by gnachman** [comment]
Oops, I merged this by accident! I've undone it.

**Pull request merged**